pymc/distributions/discrete.py

```diff
@@ -1140,17 +1140,18 @@ def rng_fn_scipy(cls, rng, lower, upper, size=None):
         return stats.randint.rvs(lower, upper + 1, size=size, random_state=rng)
 
 
-discrete_uniform = DiscreteUniformRV()
+discrete_uniform = RV()
```

Comment:
```suggestion
discrete_uniform = DiscreteUniformRV()
```

Undoing this change should fix the CI pipeline

Reply:
Thanks. That seems to be it. Not sure how that change got in there, but I'll check the diffs more carefully next time.